engine/class_modules/sc_mage.cpp

```diff
@@ -164,14 +164,14 @@ struct buff_stack_benefit_t
   const buff_t* buff;
   std::vector<benefit_t*> buff_stack_benefit;
 
-  buff_stack_benefit_t( const buff_t* _buff, const std::string& prefix ) :
+  buff_stack_benefit_t( const buff_t* _buff, util::string_view prefix ) :
     buff( _buff ),
     buff_stack_benefit()
   {
     for ( int i = 0; i <= buff->max_stack(); i++ )
     {
-      buff_stack_benefit.push_back( buff->player->get_benefit(
-        prefix + " " + buff->data().name_cstr() + " " + util::to_string( i ) ) );
+      auto benefit_name = fmt::format("{} {} {}", prefix, buff->data().name_cstr(), i);
```

Comment:
```suggestion
      auto benefit_name = fmt::format( "{} {} {}", prefix, buff->data().name_cstr(), i );
```

Reply:
Am I allowed to fully use clang-format in the mage module? I am always hesitant, since I can't remember which modules are using it and which aren't.

Reply:
We're not using clang-format at the moment